docker/deploy/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:xenial
```

Comment:
Why Ubuntu 16.04? Just curious if this was an arbitrary choice or if there are good reasons. Maybe it doesn't matter.